wp-includes/sqlite/class-wp-sqlite-pdo-engine.php

```diff
@@ -6,6 +6,7 @@
  * @since 1.0.0
  */
 
+// Require the translator class.
 require_once dirname( dirname( __DIR__ ) ) . '/lexer-explorations/class-wp-sqlite-translator.php';
```

Comment:
Should we move all the `require_once` statements to `db.php`?

Reply:
Done